indexers/vector/PysparnnIndexer/__init__.py

```diff
@@ -0,0 +1,119 @@
+from jina.executors.indexers.vector import BaseVectorIndexer
+
+
+class PysparnnIndexer(BaseVectorIndexer):
+    """
+    :class:`PysparnnIndexer` Approximate Nearest Neighbor Search for Sparse Data in Python using PySparNN.
+    """
+
+    def __init__(self, k_clusters=2, num_indexes=None, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        self.k_clusters = k_clusters
+        self.num_indexes = num_indexes
+
+    def post_init(self):
+        self.index = {}
+        self.mci = None
+
+    def _build_advanced_index(self):
+        keys = []
+        indexed_vectors = []
+        import pysparnn.cluster_index as ci
+        for key, vector in self.index.items():
+            keys.append(key)
+            indexed_vectors.append(vector)
+        
+        self.mci = ci.MultiClusterIndex(scipy.sparse.vstack(indexed_vectors), keys)
```

Comment:
`num_indexes` seems like it should be used in this `constructor`

Reply:
yes, done